src/trampoline.js

```diff
@@ -0,0 +1,46 @@
+import isFunction from './isFunction';
+
+/**
+ * A function that repeatedly calls the return value of a function,
+ *  until it is no longer a function.
+ *
+ * @func trampoline
+ * @memberOf RA
+ * @category Function
+ * @sig ... -> Any
+ * @param  {function} func
+ * @param  {} ...args
+ * @return {*} Exclude<any, Function>
+ * @example
+ *
+ * function evenOline(n) {
+ *   if (n === 0)
+ *     return true;
+ *   else
+ *     return R.partial(oddOline, [Math.abs(n) - 1]);
+ * }
+ *
+ * function oddOline(n) {
+ *   if (n === 0)
+ *     return false;
+ *   else
+ *     return R.partial(evenOline, [Math.abs(n) - 1]);
+ * }
+ *
+ * RA.trampoline(oddOline, 3); //=> true
+ * RA.trampoline(evenOline, 200000); //=> true
+ * RA.trampoline(oddOline, 300000); //=> false
+ * RA.trampoline(evenOline, 200000000); //=> true
+ */
+
+const trampoline = (func, ...args) => {
```

Comment:
Every function in `RA` must be curried. So we have to transform current signature into following:
```js
const trampoline = curry((fn, args) =>
 ...body...
);
```
Please compensate this change in all tests, typings and typing tests.